build/includes/sdk.mk

```diff
@@ -137,7 +137,6 @@ run-sdk-conformance-no-build: ensure-build-sdk-image
 run-sdk-conformance-test:
 	$(MAKE) run-sdk-command COMMAND=build-sdk-test
 	$(MAKE) run-sdk-conformance-no-build
-	$(MAKE) run-sdk-command COMMAND=clean
```

Comment:
Note that there is also a clean step in Rust. Will check if Rust package files are gitignored.
I suggest adding new make target for clean.

Reply:
Rust target dir have appropriate .gitignore. https://github.com/googleforgames/agones/blob/master/test/sdk/rust/.gitignore
So the only request for adding new target for clean left.

Could be:
```
clean-sdk-conformance-tests:
	$(MAKE) run-all-sdk-command COMMAND=clean
```

Reply:
Created a PR linked to this.